Update Data sharing and archiving in the RDM book with a Data Availability Statement section and fix duplication with Open Data section (#1764)
alan-turing-institute/the-turing-way: book/website/reproducible-research/open/open-data.md

Any user is permitted to download, copy, analyse, re-process, and re-use it for any other purpose with minimal financial, legal, and technical barriers.

This represents a real shift in how research works. Funders are starting to require researchers to make their data available and submit data management plans {ref}`Data Management Plans<rr-rdm-dmp>` as part of project proposals.
At the moment, anyone who wishes to use data from a researcher often has to contact that researcher and make a request.
"Open by default" remedies this with a presumption of publication for all.
If access to data is restricted, for instance, due to security reasons, the justification for this should be made clear.
Free access to, and subsequent use of, data is of significant value to society and the economy. 
That data should, therefore, be open by default and only as closed as necessairy. 

You can find more about the practical steps to make your data available in the {ref}`Steps to Share your Data <rr-rdm-sharing-steps>` in the {ref}`Sharing and Archiving Data section <rr-rdm-sharing>`

(rr-open-data-barriers)=
## Barriers to Data Sharing
Some academics still find sharing data difficult.

> Recent surveys {cite}`Stuart2018sharing` conducted amongst researchers show that they find sharing data difficult because:

Reviewer: ```suggestion
Recent surveys {cite}`Stuart2018sharing` conducted amongst researchers list the following reasons:
```

- Organising data in a presentable and useful way is challenging (mentioned by 46%)

> - They are unsure about copyright and licensing (mentioned by 37%)

Reviewer: ```suggestion
- Researchers are unsure about copyright and licensing (mentioned by 37%)
```

> - They do not know which repository to use as (raised by 33%)

Reviewer: ```suggestion
- Researchers do not know which repository to use for different data types (raised by 33%)
```

These are cultural challenges that might be addressed in changing practice going forward.
However, there are also legal, ethical or contractual reasons that sometimes prevent making data publicly available in its entirety or even in parts.

> Here are some reasons why this may be the case:

Reviewer: ```suggestion
Below, we discuss some reasons explaining why this may be the case.
```
